src/plugin_discovery/find_plugin_specs.js

```diff
@@ -174,7 +197,7 @@ export function findPluginSpecs(settings, config = defaultConfig(settings)) {
     // only emitted once it is fully extended by all
     extendedConfig$: extendConfig$
       .ignoreElements()
-      .concat([config]),
```

Comment:
The purpose of `.concat()` here was to wait until `extendConfig$` completes and `config$` is fully extended before emitting. If the idea here is to subscribe to `config$` before `extendConfig$` completes, perhaps `.combineLatest(extendConfig$.last(), config$).map(x => x[1])` is more what we want.

Reply:
What if we just emit the config from the extendConfig$? I pushed a commit that does just this, lemme know what you think.